src/Microsoft.IdentityModel.Logging/SecurityArtifact.cs

```diff
@@ -14,44 +14,100 @@ namespace Microsoft.IdentityModel.Logging
     /// </remarks>
     internal struct SecurityArtifact : ISafeLogSecurityArtifact
     {
+        private const string _scrubbedArtifact = "#ScrubbedArtifact#";
```

Comment:
Should we have some info here, similar to our PII string with a link to a wiki explaining how to control this feature?

Reply:
Wrong level of abstraction, that's only relevant in the logging library.

Reply:
What will users see in the log?